src/libraries/Microsoft.Extensions.Caching.Abstractions/ref/Microsoft.Extensions.Caching.Abstractions.Typeforwards.netcoreapp.cs

```diff
@@ -0,0 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// The compiler emits a reference to the internal copy of this type in our non-NETCoreApp assembly
+// so we must include a forward to be compatible with libraries compiled against non-NETCoreApp Microsoft.Extensions.Caching.Abstractions
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.CompilerServices.IsExternalInit))]
```

Comment:
We shipped a public System.Runtime.CompilerServices.IsExternalInit in Microsoft.Extensions.Caching.Abstractions?

Reply:
No. But once you use `init` in a `netstandard2.0` library, and someone else references your library, Roslyn binds to the `IsExternalInitAttribute` in your assembly.

See discussion starting at https://github.com/dotnet/runtime/issues/61737#issuecomment-974271053